app/celery/scheduled_tasks.py

```diff
@@ -207,14 +209,38 @@ def replay_created_notifications():
 
 @notify_celery.task(name='check-if-letters-still-pending-virus-check')
 def check_if_letters_still_pending_virus_check():
-    letters = dao_precompiled_letters_still_pending_virus_check()
+    letters = []
+
+    for letter in dao_precompiled_letters_still_pending_virus_check():
+        # find letter in the scan bucket
+        filename = generate_letter_pdf_filename(
+            letter.reference,
+            letter.created_at,
+            ignore_folder=True,
+            postage=letter.postage
+        )
+
+        if s3.file_exists(current_app.config['LETTERS_SCAN_BUCKET_NAME'], filename):
+            current_app.logger.warning(
+                f'Letter id {letter.id} got stuck in pending-virus-check. Sending off for scan again.'
+            )
+            notify_celery.send_task(
+                name=TaskNames.SCAN_FILE,
+                kwargs={'filename': filename},
+                queue=QueueNames.ANTIVIRUS,
+            )
+        else:
+            letters.append(letter)
 
     if len(letters) > 0:
         letter_ids = [(str(letter.id), letter.reference) for letter in letters]
 
-        msg = """{} precompiled letters have been pending-virus-check for over 90 minutes. Follow runbook to resolve:
+        msg = f"""{len(letters)} precompiled letters have been pending-virus-check for over 90 minutes.
+            We couldn't find them in the scan bucket.
+
+            Follow runbook to resolve:
             https://github.com/alphagov/notifications-manuals/wiki/Support-Runbook#Deal-with-letter-pending-virus-scan-for-90-minutes.
```

Comment:
This run book entry will need to be updated.

Reply:
Yeah, it was on my radar to do when merging this.

That said, your comment prompted me to do some thinking about it and drafting up new words - and now I'm in favour of deleting the runbook. I don't think I can offer any concrete steps that we know you'll have to do when the letter is in `pending-virus-scan` but ISN'T in the scan bucket. I don't think we need a runbook just to say "you will have to investigate and find the letter and may need to run any number of different tasks to unstick this".

Reply:
Deleting the entry is probably best.

Reply:
Shall we remove the reference to it here?

Reply:
Yeah lets delete it.